output.tf

```diff
@@ -1,3 +1,7 @@
 output "key_name" {
-  value = "${aws_key_pair.default.key_name}"
+  value = "${aws_key_pair.non_empty.key_name}"
+}
+
+output "key_name_generated" {
```

Comment:
We do not need this output. We want a single output as that will make integration easier.